docs/workflows/collections.rst

```diff
@@ -136,6 +136,30 @@ Remote GET Response::
         "url": "https://galaxy-dev.ansible.com/api/v2/collections/testing/ansible_testing_content/",
     }
 
+For content sources token authentication. You can provide the ``token`` and/or ``auth_url``.
```

Comment:
```suggestion
For remote sources that require authentication, tokens can be used. You can provide the ``token`` and/or ``auth_url``.
```

^ still needs line wrapping

Also finding a way to https://docs.ansible.com/ansible/latest/user_guide/collections_using.html#configuring-the-ansible-galaxy-client here would be good too.